development/data-flows-for-analytics.md

```diff
@@ -0,0 +1,85 @@
+# Data flows
+
+In this page we focus on how data flows through the various CHT components. CHT is built to support the delivery of quality community health in hardest to reach areas. It is therefore designed to work in areas with low connectivity making it an offline-first toolkit. The architectural and technology choices in the stack are mostly guided by this principle which will be evident in the discussion of the data management pipeline.
+
+
+## Overview
+
+![overview](img/data-flows-overview-1.png)
+
+![overview](img/data-flows-overview-2.png)
+
+At a high level:
+
+- data is collected from the device of a health worker;
+- this data is pushed to an online instance from where this data is available to other health workers, supervisors and decision makers;
+- the data is transferred to a relational database and presented in forms suitable for analytics and visualizations;
+- more access to the data is given to relevant parties at this level, for example analysts;
+- managers and decision makers access the same from visualizations of the data.
```

Comment:
Maybe include a line stating that visualization platforms can then be connected to PostgreSQL